### DISCUSSION TITLE
Unwanted border added to images in published content

### DISCUSSION THREAD

**Discussion by JaniceManwiller** [GitbookIO/community#830] [0. General bugs and questions] [answered]
In the published content, it looks like GitBook is adding a border to all of the images. This isn't as noticeable in most screenshots, which are rectangular to begin with, but is very distracting for diagrams.

For example, here is a diagram image in GitBook:

![image](https://github.com/user-attachments/assets/4b5ca310-95f9-49d0-b9eb-f5f0392750c5)

And here is the same image in the published version:

![image](https://github.com/user-attachments/assets/315da99c-d444-4abc-a49a-c509d5cc8f25)

I wouldn't object to borders if they were configurable, but there doesn't seem to be any way to remove them.

**Comment by viktorrenkema** [comment, marked as answer]
Hey Janice! Good point, I agree that it hurts more than that it offers. I've looked into why this is and made a PR to remove the border: https://github.com/GitbookIO/gitbook/pull/2742
